menas/src/main/scala/za/co/absa/enceladus/menas/services/AttachmentService.scala

```diff
@@ -36,8 +36,8 @@ class AttachmentService @Autowired()(attachmentMongoRepository: AttachmentMongoR
   def uploadAttachment(attachment: MenasAttachment): Future[Completed] = {
     chooseRepository(attachment.refCollection).getLatestVersionValue(attachment.refName).flatMap {
       case Some(version) =>
-        val updated = attachment.copy(refVersion = version + 1)
-        attachmentMongoRepository.create(updated)
+        // the attachment version should already increased by one from the currently latest by the caller
```

Comment:
`version` on the line above not used anymore, also

```suggestion
        // the attachment version should already be increased by one from the currently latest by the caller
```

Reply:
Thanks, fixed.